tests/test_constants.py

```diff
@@ -7,6 +8,21 @@
 
 
 class TestConstants(unittest.TestCase):
+    @staticmethod
+    def _exec(command):
+        process = os.popen(command)
+        output = process.read().strip()
+        process.close()
+        return output
+
+    @staticmethod
+    def _extract_current_app(dumpsys_output):
```

Comment:
Should we rename this (and all other occurrences) to `_parse_current_app` to properly reflect the constant being tested?

Reply:
```suggestion
    def _current_app(dumpsys_output):
```

Another change suggestion to follow...